apps/expo/src/components/rich-text.tsx

```diff
@@ -58,7 +58,11 @@ export const RichText = ({ text, facets, size = "base" }: Props) => {
       } else {
         parts.push({
           text: segment.text,
-          component: segment.text,
+          component: (
+            <Text className="text-neutral-500 dark:text-neutral-50">
+              {segment.text}
+            </Text>
+          ),
```

Comment:
Text is way too light on light mode. I'd be inclined to removed all instances of `text-neutral-500` and just use the automatic colour